helm/ako/values.yaml

```diff
@@ -40,11 +40,10 @@ NetworkSettings:
   #       - 11.0.0.1/24
   subnetIP: "" # Subnet IP of the vip network
   subnetPrefix: "" # Subnet Prefix of the vip network
-  networkName: "" # Network Name of the vip network
   enableRHI: false # This is a cluster wide setting for BGP peering.
-  vipNetworkList: []
-  # vipNetworkList: []
-  #  - networkName: subnet-xxxxxxx
+  vipNetworkList: [] # Network Name of the VIP network, multiple networks allowed only for AWS Cloud.
```

Comment:
@monotosh-avi can we add somewhere in the documentation that we will take the default VIP n/w (IPAM) if the `vipNetworkList` is empty?

Reply:
Sure. I'll add.